packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -227,7 +228,7 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
             } else if (/proof-of-address/.test(route_path) || /proof-of-identity/.test(route_path)) {
                 return !should_allow_authentication;
             } else if (/proof-of-ownership/.test(route_path)) {
-                return is_virtual;
+                return is_virtual || !is_pending_proof_of_ownership;
```

Comment:
just a question : 
don't we need to add this condition to desktop menu as well?

Reply:
@shayan-deriv, This is added in the `account.jsx` file 😄 Is there another place you think I should add it to?

Reply:
oh.u're right. I guess I missed that.